techniques/html/H100.html

```diff
@@ -107,15 +107,15 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
     
     
     <section id="description">
-      <h2>Description</h2>
-      <p>The objective of this technique is to provide examples of properly marked up email and password inputs. This technique involves providing form mechanism where the user can enter their email address and password to log into the website.</p>
-      <p>Browsers and password managers generally use the accessible name of the input to determine how to fill it in. The user agent (including password managers) should not be blocked from populating the fields, and pasting by the user should also not be blocked.</p>
+      <h2>解説</h2>
+      <p>このテクニックの目的は、適切にマークアップされた電子メール及びパスワードの入力の例を提供することである。このテクニックでは、利用者がメールアドレス及びパスワードを記入してウェブサイトにログインできるフォームメカニズムを提供する。</p>
+      <p>ブラウザ及びパスワードマネージャーは通常、入力のアクセシブルな名前 (name) を使用して記入方法を決定する。ユーザエージェント (パスワードマネージャーを含む) によるフィールドへの入力をブロックすべきではなく、利用者による貼り付けもブロックすべきではない。</p>
     </section>
     <section id="examples">
-      <h2>Examples</h2>
+      <h2>事例</h2>
       <section class="example" id="example-1-properly-marked-up-email-and-password-inputs">
-        <h3>Example 1: properly marked up email and password inputs</h3>
-        <p>A secure website has an email and password based login form.</p>
+        <h3>事例 1: 適切にマークアップされた電子メール及びパスワードの入力</h3>
+        <p>安全なウェブサイトには、電子メール及びパスワードに基づくログインフォームがある。</p>
```

Comment:
ここでの secure ってどういう意味なんでしょうね。意味の広い言葉ですが、ここでは単に「認証がある」という意味で使っているような気がします。こんな感じではないでしょうか。

```suggestion
        <p>認証が必要なウェブサイトがあり、電子メール及びパスワードに基づくログインフォームがある。</p>
```